satpy/etc/readers/hdfeos_l1b.yaml

```diff
@@ -488,7 +488,8 @@ file_types:
     - 'thin_M{platform_indicator:1s}D021KM.A{start_time:%Y%j.%H%M}.{collection:03d}{suffix}.hdf'
     - 'M{platform_indicator:1s}D021KM.{start_time:%y%j%H%M%S}.hdf'
     - '{platform_indicator:1s}1.{start_time:%y%j.%H%M}.1000m.hdf'
-    file_reader: !!python/name:satpy.readers.hdfeos_l1b.HDFEOSBandReader
+    - 'M{platform_indicator:1s}D021KM_A{start_time:%Y%j_%H%M}_{collection:03d}_NRT.hdf'
```

Comment:
Isn't this pattern already handled by the pattern on line 487?

Reply:
No, it's underscores now, not dots...